src/main/kotlin/com/jetbrains/snakecharm/inspections/SmkUnrecognizedSectionInspection.kt

```diff
@@ -31,7 +31,7 @@ class SmkUnrecognizedSectionInspection : SnakemakeInspection() {
                     sectionNamePsi,
                     SnakemakeBundle.message("INSP.NAME.section.unrecognized.message", sectionKeyword),
                     ProblemHighlightType.WEAK_WARNING,
-                    null, AddIgnoredElementQuickFix(sectionNamePsi)
+                    null, AddIgnoredElementQuickFix(sectionNamePsi.text)
```

Comment:
The section name is already stored in `sectionKeyword` variable, please use it instead